src/structs/lepton_header.rs

```diff
@@ -68,11 +70,16 @@ pub struct LeptonHeader {
 
     pub jpeg_file_size: u32,
 
-    /// on decompression, plain-text size
-    pub plain_text_size: u32,
+    /// on decompression, uncompressed lepton header size. This is only
+    /// saved by this encoder for historical reasons. It is not used be
+    /// the decoder.
+    pub uncompressed_lepton_header_size: Option<u32>,
+
+    /// the git revision of the encoder that created this file (first 8 hex characters)
+    pub git_revision_prefix: [u8; 4],
```

Comment:
Not sure this is the best way to go. Why not classical versioning scheme?

Reply:
Don't want to break backward compat with the flags